src/dmd/json.d

```diff
@@ -867,9 +962,8 @@ public:
         foreach (m; Module.amodules)
         {
             objectStart();
-            if(m.md)
-                property("name", m.md.toChars());
-            property("file", m.srcfile.toChars());
+            propertyOrNull("name", m.md ? m.md.toChars() : null);
+            propertyOrNull("file", m.srcfile.toChars());
```

Comment:
Can this even be `null`?

Reply:
no, but it seemed like consistently using `propertyOrNull` was better.  The new objects should use `propertyOrNull` and the old `modules` object uses property....maybe there is a better name....

Reply:
The name `requiredProperty` came to mind and seems like a good idea.